policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/phase_out/percentage.yaml

```
@@ -0,0 +1,15 @@
+description: Rhode Island phases the standard deduction out at this rate, depending on the increments their income exceeds the threshold.
```

We usually use `depending on` to refer to the `breakdown`
```suggestion
description: Rhode Island phases out this percentage of its standard deduction for each increment by which their income exceeds the threshold.
```